code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -1380,6 +1380,9 @@ datum
 
 			on_mob_life(var/mob/M, var/mult = 1) // allergies suck fyi
 				if (!M) M = holder.my_atom
+				if(M.reagents?.has_reagent("antihistamine") && prob(66)) //66% (intentionally not probmult) chance to cancel standard histamine effects, including hyperallergenic hist duplication
```

Comment:
`66% (intentionally not probmult)`
why?

Reply:
because it's mutually exclusive events: either it returns early (2/3) or it does not (1/3)